src/game/Battle.java

```diff
@@ -23,8 +28,39 @@ public int getTime() {
     }
 
     public void nextMoment() {
-        ship1.calculateState();
-        ship2.calculateState();
+
+
         time++;
```

Comment:
Metody które wykonują jedno proste wyrażenie mogą być zbędne. Dokładają tylko dodatkowej abstrakcji - programiści muszą zastanawiać się co oznacza "nextMoment", a kryje się za tym jedynie logika podniesienia czasu o 1. Nie ma sensu tutaj korzystać z metody, bo robi ona tylko jedną, bardzo prostą akcję, i zmienia tej akcji nazwę. Lepiej zawrzeć to w jakiejś większej metodzie robiącej od razu kilka rzeczy.